tools/js-sdk-release-tools/src/llc/apiVersion/apiVersionTypeExtractor.ts

```diff
@@ -57,17 +57,33 @@ export const getApiVersionType: IApiVersionTypeExtractor = async (
         if (typeFromClient !== ApiVersionType.None) return typeFromClient;
     }
 
+    const isModelOnlyPackage = await isModelOnly(packageRoot);
+    if (isModelOnlyPackage) {
+        logger.info('No operation found, fallback to get api version type from latest version in NPM');
+        const packageName = getNpmPackageName(packageRoot);
```

Comment:
extract to a function into `xlc/apiVesion/utils.ts` and reuse. I remember it is used in multiple places.